packages/core/src/command/hl7v2-subscriptions/hl7-subscriptions-sftp-ingestion/hl7-notification-sftp-ingestion-factory.ts

```diff
@@ -0,0 +1,13 @@
+import { Config } from "../../../util/config";
+import { Hl7SubscriptionLaHieIngestionCloud } from "./hl7-notification-sftp-ingestion-cloud";
+import { Hl7SubscriptionLaHieIngestionDirect } from "./hl7-notification-sftp-ingestion-direct";
+import { Hl7SubscriptionLaHieIngestion, LaHieSftpClient } from "./hl7-subscriptions-sftp-ingestion";
+
+export async function buildLaHieIngestionDirect(): Promise<Hl7SubscriptionLaHieIngestion> {
```

Comment:
this isn't the direct implementation - this is the factory (see the pattern / another example)